CONTRIBUTING.md

```diff
@@ -191,6 +191,43 @@ A simple way to use this on a laptop is the following:
 For example if the test blocks, it will show the line in question with `block_on`.
 See the documentation of `tokio-console` for more details.
 
+### Tracking metrics
+
+The prometheus system is used for keeping track of the validator in a Kubernetes setting.
+However, this can also be used locally to run test case and then get metrics to work on.
+This is a little bit more indirect than having runtimes in log file, but works fine.
+The steps are the following:
+
+1. The list of created metric server can be seen in the validator logs with the entries "Starting to serve metrics".
```

Comment:
`The list of the metric service ports` ?

Reply:
Ok.